.github/workflows/tests.yml

```diff
@@ -17,6 +21,7 @@ jobs:
     runs-on: ubuntu-latest
     container: returntocorp/ocaml:alpine-2022-06-09
     steps:
+      #TODO: why do we need this? What does not work if we don't have this?
```

Comment:
This was needed the last time I looked at this part of the workflow (~3 months ago). Maybe @underyx knows.

Reply:
I think @mjambon actually wrote those pre-checkout fixes